added the group norms to the collaboration readme

ZaidMazen1 opened the pull request:
Added the group norms we agreed on in the document to the collaboration readme.

ZaidMazen1 committed:
added the group norms to the collaboration readme

Ammmoe commented on collaboration/README.md:
```diff
@@ -1,5 +1,19 @@
+## Summary of Group Norms
 
+    We are mature, respectful, and trust one another to be consistently trying
+    their best to contribute to the group. We also hope for thegroup to stay 
```
Hi Zaid!
Please add a space between 'the' and 'group' in this line.

ZaidMazen1 committed:
fixed small grammar mistake

Ammmoe committed:
Update settings.json

I change the lines 122 and 123 from "explicit" to "true" because it is set to "true" in the upstream.

Ammmoe committed:
Update settings.json

Ammmoe reviewed (approved):
Thank you for adding the **`group norms`** and the writing covers what we discussed during our initial meetings. The code also passes all CI checks so great job!

KarimMakki reviewed (approved):
Thanks @ZaidMazen1 for doing your task and adding the group norms to the collaboration readme! looks great
